### PR TITLE
Consistent style in unit tests

### PR DESCRIPTION
**Pull request by Alex7Li**
Fix style inconsistencies in the unit test code. The following were checked:
1) Use arrow functions whenever applicable.
2) Capitalize only the sentence in the outer describe block.
3) The first sentence of the outer describe block says "The  '(function name)' function  of (helper/plain)"
4) Use jasmine specific matchers toBeTrue/False/Undefined/Null when you can
5) Add parameter name comments to the first function of a file if it is used a billion times and isn't clear.

### CODE REVIEW

File: test/hasOwn_test.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe('the hasOwn method', function() {
     assertHasOwn({a: 1}, 'constructor', false);
   });
 
-  it('recognizes function properties', () => {
+  xit('recognizes function properties', () => {
```
Comment (kjgalvan) [inline test/hasOwn_test.js:53]: Is this change intentional?

Comment (Alex7Li) [reply test/hasOwn_test.js:53]: No

File: test/get_test.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ goog.setTestOnly();
 
 const {DataLayerHelper} = goog.require('helper');
 
-describe('The helper.get function', function() {
+describe(`The 'get' function of helper`, () => {
```
Comment (durbanie) [inline test/get_test.js:6]: nit: I'd prefer to use single quotes for the full description, and ticks for the names of functions/methods within the description. Same everywhere else.

File: test/get_test.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ goog.setTestOnly();
...
   const h = new DataLayerHelper([{
```
Comment (durbanie) [inline test/get_test.js:7]: Not your change, but just noticed this. Can you add a parameter name comment?
https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments

Can be in a separate PR. It would be good to audit all tests for this.

File: test/hasOwn_test.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ goog.setTestOnly();
 
 const {hasOwn} = goog.require('plain');
 
-describe('the hasOwn method', function() {
+describe(`The 'hasOwn' method of plain`, function() {
```
Comment (durbanie) [inline test/hasOwn_test.js:6]: Switch to an arrow function?

File: test/merge_test.js [hunk 5]
```diff
@@ -51,22 +51,21 @@ describe('The merge function', function() {
...
-        assertMerge({a: {}}, {a: 1, b: 2}, {a: {}, b: 2});
-        assertMerge({a: {x: 1, y: 2}}, {}, {a: {x: 1, y: 2}});
-        assertMerge({a: {x: 1, y: 2}}, {a: 1}, {a: {x: 1, y: 2}});
-        assertMerge({a: {x: 1, y: 2}}, {b: 2}, {a: {x: 1, y: 2}, b: 2});
-        assertMerge({a: {x: 1, y: 2}}, {a: 1, b: 2}, {a: {x: 1, y: 2}, b: 2});
-        assertMerge({a: {}, b: {}}, {}, {a: {}, b: {}});
-        assertMerge({a: {}, b: {}}, {a: 1, b: 2}, {a: {}, b: {}});
-        assertMerge({a: {}, b: {}}, {a: 1, b: 2, c: 3}, {a: {}, b: {}, c: 3});
-      });
+  it(`overwrites other type objects when merging with a plain object`, () => {
```
Comment (durbanie) [inline test/merge_test.js:54]: Single quotes?

Comment (Alex7Li) [reply test/merge_test.js:54]: I'm so impressed that you noticed that

File: test/get_test.js (hunk 3 above)
Comment (Alex7Li) [reply test/get_test.js:7]: I added the comment to this file, but didn't see anything in the others.

Comment (Alex7Li) [reply test/get_test.js]: There are some methods that could benefit from this. Since the same function is just used again and again tens of times, I  only put the comment in the first function of any file
